Cargo.toml

```
@@ -34,6 +34,7 @@ simple-eyre = "0.3.1"
 tokio = { version = "1.27.0", features = ["rt-multi-thread", "macros"] }
 basic-toml = "0.1.2"
 url = "2.3.1"
+mime_guess = "2.0.4"
```

We don't need the reverse mappings so can turn that feature off and avoid some unused code.

```suggestion
mime_guess = { version = "2.0.4", default-features = false }
```

Also it should be in the list after the `log` crate. (It looks like `basic-toml` is in the wrong place, which makes `cargo add` not treat the crate list as ordered)

---

I ordered the list